drift-client/src/main/java/com/facebook/drift/client/DriftMethodInvocation.java

```diff
@@ -330,7 +332,11 @@ private synchronized void fail(String reason)
                 invocationAttempts,
                 succinctNanos(ticker.read() - startTime),
                 failedConnections,
-                overloadedRejects);
+                overloadedRejects,
+                attemptedAddresses.stream()
+                        .map(Address::getHostAndPort)
+                        .map(HostAndPort::getHost)
```

Comment:
Let's don't drop the port. It is possible that 2 or more services are running on the same host.